### PR TITLE
Updated Machine ID documentation.

### PR DESCRIPTION
**Pull request by russjones**

### CODE REVIEW

File: docs/pages/machine-id/guides/jenkins.mdx [hunk 1]
```diff
@@ -54,6 +68,10 @@ scope for server access, reduce the blast radius if one pipeline is
 compromised, and allow you to remotely audit and lock pipelines if you detect
 malicious behavior.
 
+<Figure align="left" bordered caption="Jenkins Deployments">
+  ![Jenkins Deployments](../../../img/machine-id/jenkins.png)
+</Figure>
```
Comment (zmb3) [inline docs/pages/machine-id/guides/jenkins.mdx:73]: Should this be a separate commit? This way you can just revert the commit with the admonitions when we release 9.1.

File: docs/pages/machine-id/guides/jenkins.mdx [hunk 2]
```diff
@@ -19,6 +19,20 @@ You will need the following tools to use Teleport with Jenkins.
...
 - The `tctl` admin tool version >= (=teleport.version=)
 
+<Admonition type="note" title="Machine ID and TLS Routing">
+TLS Routing support will be added to Machine ID in [Teleport
+9.1](https://goteleport.com/docs/preview/upcoming-releases/#teleport-91). Until
+that time, the Teleport Proxy Server will need to be configured with a
+dedicated SSH listener.
+
+```yaml
+proxy_service:
```
Comment (zmb3) [inline docs/pages/machine-id/guides/jenkins.mdx:29]: Don't you also need to ensure version is not v2?

File: docs/pages/machine-id/guides/jenkins.mdx (hunk 1 above)
Comment (russjones) [reply docs/pages/machine-id/guides/jenkins.mdx:74]: Good point, fixed.

You might already know this, but when you want to split changes like this into separate commits, you don't need to copy them over to a temporary file then copy them back. You can do a `get reset HEAD~1` then added the parts of `jenksin.mdx` sepearly  `git add -p`.

**Review by Valien — APPROVED** [review]

**Review by zmb3 — APPROVED** [review]
